package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "4.1.8+20201218.2109",
+    "version": "4.1.8.1+20201222.1600",
```

Comment:
I'm still not fine on the use of semver here. But again, @jxn-30 proposed the usage of this.